src/components/Button/test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { renderWithClient } from 'testUtils';
+import { Button, ButtonUI } from '.';
+
+describe('ButtonUI', () => {
```

Comment:
I keep meaning to take out the separate UI tests, I will do that now, but you can remove this one.